pkg/skaffold/build/scheduler.go

```diff
@@ -112,7 +113,12 @@ func (s *scheduler) build(ctx context.Context, tags tag.ImageTags, i int) error
 	finalTag, err := performBuild(ctx, w, tags, a, s.artifactBuilder)
 	if err != nil {
 		event.BuildFailed(a.ImageName, err)
-		eventV2.BuildFailed(a.ImageName, err)
+		if ctx.Err() == context.Canceled {
```

Comment:
use `errors.Is(ctx.Err(), context.Canceled)` instead

Reply:
oops, thanks for pointing this out